sdk/sdk-view/src/test/java/mil/emp3/api/utils/ComparisonUtils.java

```diff
@@ -0,0 +1,347 @@
+package mil.emp3.api.utils;
+
+import android.graphics.Bitmap;
+
+import org.cmapi.primitives.GeoIconStyle;
+import org.cmapi.primitives.GeoPoint;
+import org.cmapi.primitives.GeoPolygon;
+import org.cmapi.primitives.GeoPosition;
+import org.cmapi.primitives.GeoRectangle;
+import org.cmapi.primitives.IGeoAltitudeMode;
+import org.cmapi.primitives.IGeoColor;
+import org.cmapi.primitives.IGeoFillStyle;
+import org.cmapi.primitives.IGeoIconStyle;
+import org.cmapi.primitives.IGeoLabelStyle;
+import org.cmapi.primitives.IGeoPoint;
+import org.cmapi.primitives.IGeoPolygon;
+import org.cmapi.primitives.IGeoPosition;
+import org.cmapi.primitives.IGeoRectangle;
+import org.cmapi.primitives.IGeoStrokeStyle;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+
+import mil.emp3.api.Point;
+import mil.emp3.api.Polygon;
+import mil.emp3.api.Rectangle;
+import mil.emp3.api.enums.FeatureTypeEnum;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+
+/**
+ * Created by matt.miller@rgi-corp.local on 10/27/17.
+ */
+
+public class ComparisonUtils {
```

Comment:
doesn't this class already exist? Maybe all the code repeat is due to not updating your branch with development.  It looks like you have added a lot more than you have